suite-native/intl/src/messages.ts

```diff
@@ -2524,6 +2524,19 @@ export const messages = {
             updateToVersionAvailable: 'Update to version {firmwareVersion} available',
             updateButton: 'Update',
         },
+
+        firmwareInfoScreen: {
+            title: 'Firmware update',
+            subtitle: "Firmware is your Trezor's operating system",
+            list: {
+                item1: 'This update will take some time to complete.',
+                item2: '<b>Don’t leave or close the app</b> during installation. Doing so will corrupt the firmware.',
+                item3: 'While the firmware is updating, <b>leave your phone as is</b>. It won’t shut off.',
```

Comment:
I see `While the firmware is installing` for item3 in the Figma. Not sure about the item1, it might be better to use `installation` instead of `update` too. I believe it's more universal and works fine for both FW update and installation.